javascript/ql/src/Security/CWE-347/MissingJWTKeyVerification.ql

```diff
@@ -10,12 +10,11 @@
  */
 
 import javascript
-import DataFlow
 import semmle.javascript.RestrictedLocations
 
-from CallNode call
+from DataFlow::CallNode call
 where
-  call = moduleMember("jsonwebtoken", "verify").getACall() and
+  call = DataFlow::moduleMember("jsonwebtoken", "verify").getACall() and
   unique(boolean b | b = call.getArgument(1).analyze().getABooleanValue()) = false
```

Comment:
This should be used instead: 
https://github.com/github/codeql/blob/ee52774e9079562ed30ce9998707f120de6192cb/javascript/ql/lib/semmle/javascript/dataflow/TypeInference.qll#L99-L100